app/views/shared/_bottom_cta.html.erb

```diff
@@ -1,5 +1,7 @@
 <div class="py-20 text-center max-w-4xl mx-auto">
   <h4 class="text-2xl mb-4 font-light leading-10 text-gray-600 dark:text-gray-200"><%= heading %></h4>
+  <% if defined?sub_heading %>
```

Comment:
Nit: Rails partials have a [local_assigns](https://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables) hash we can use for this kind of thing.

```suggestion
  <% if local_assigns[:sub_heading].present? %>
```